spec/controllers/hyrax/admin/admin_sets_controller_spec.rb

```diff
@@ -184,6 +184,20 @@
           expect(AdminSet.exists?(admin_set.id)).to be true
         end
       end
+
+      context "with a referer" do
+        before do
+          request.env['HTTP_REFERER'] = 'http://test.host/foo'
```

Comment:
why is the referrer important here?

Reply:
Okay, if it is not needed, I'll remove it. I was following the pattern from: spec/controllers/hyrax/dashboard/collections_controller_spec.rb
Thanks for pointing that out.